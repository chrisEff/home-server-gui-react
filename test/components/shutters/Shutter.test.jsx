const React = require('react')
const Shutter = require('../../../src/components/shutters/Shutter')

const {shallow} = require('enzyme')

describe('Shutter', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<Shutter shutter={{
			id: 1,
			name: 'Office',
			codeDown: 12345678,
			codeUp: 87654321,
			protocol: 4,
		}} />)
		expect(wrapper).toMatchSnapshot()
	})
})
