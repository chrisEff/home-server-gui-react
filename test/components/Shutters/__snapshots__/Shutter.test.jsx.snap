// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Shutter should render correctly 1`] = `
<div
  className="device_n9mjiu-o_O-shutter_zf5bi4"
>
  <span>
    Office
  </span>
  <br />
  <span
    className="button_j63qv8"
    onClick={[Function]}
  >
    ⬆️
  </span>
  <br />
  <span
    className="button_j63qv8"
    onClick={[Function]}
  >
    ⬇️
  </span>
</div>
`;
